test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests.cs

```diff
@@ -1416,6 +1417,215 @@ public async Task RestoreCommand_CentralVersion_ErrorWhenCentralPackageVersionFi
             Assert.True(errorMessage.Contains(autoreferencedpackageId));
         }
 
+        [Fact]
+        public async Task RestoreCommand_LogDowngradeWarningsOrErrorsAsync_ErrorWhenCpvmEnabled()
+        {
+            // Arange
+            // create graph with a downgrade
+            string centralPackageName = "D";
```

Comment:
I don't want to be too picky on tests but fwiw, this is not in line with our guideline
See 10 from https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md

Reply:
Updated